src/models/quotations/Quotation.js

```diff
@@ -74,6 +76,22 @@ class QuotationService extends BaseModel {
     return new this.axios.post(url, {}).then(response => response.data)
   }
 
+  generatePdf(id) {
+    const url = `/quotation/quotation/${id}/generate_definitive_pdf/`
+    return new this.axios.post(url, {}).then(response => response.data)
+  }
+
+  async downloadPdf(id) {
+    const url = `/quotation/quotation/${id}/download_definitive_pdf/`
+    const token = await this.getCsrfToken()
+    const headers = this.getHeaders(token)
+    headers.responseType = 'blob'
```

Comment:
we have a generic method for downloading which can directly be used in the view https://github.com/My24Service/frontend/blob/848f3f2607cbb58aedf4caed3e470dd6372fd20c/src/services/my24.js#L23